02-express-tutorial/app.js

```diff
@@ -1 +1,159 @@
-console.log('Express Tutorial')
+const express = require("express");
+const app = express();
+const people = require('./routes/people')
+const authRoutes = require('./routes/auth')
+const cookieParser = require('cookie-parser');
+
+
+//static assets
+app.use(express.static("./methods-public"));
+//pase from data
+app.use(express.urlencoded({ extended: false }));
+// parse json
+app.use(express.json());
+
+// parse cookies
+app.use(cookieParser());
+
+//authentication routes under the '/logon' prefix
+app.use("/auth", authRoutes);
+
+
+app.use('/api/people',people)
```

Comment:
Welldone so far, this is a clean codebase. Be consistent with naming, and choose a descriptive naming convention, specifying the actor's name (in this case, 'people') is typically meant for the model (schema) of the actor, but here, you are specifying the route, then you should use the name (people) and function it performs (route) just as you've done for auth (authRoutes).